docs/components/connectors/out-of-the-box-connectors/amazon-dynamodb.md

```diff
@@ -55,7 +55,9 @@ Choose one of the following methods:
 
 ## Authentication
 
-To access the **Amazon DynamoDB service**, the connector needs the appropriate credentials. The following authentication options are available:
+Choose an applicable authentication type from the `Authentication` dropdown. Learn more about authentication types in the related [appendix entry](#aws-authentication-types).
+
+If chosen `credentials`, to access the **Amazon DynamoDB service**, the connector needs the appropriate credentials. The following authentication options are available:
```

Comment:
Maybe we can remove the other auth related content here as well and always refer to our AWS auth guide?

```
If chosen `credentials`, to access the **Amazon DynamoDB service**, the connector needs the appropriate credentials. The following authentication options are available:

- **Access Key**: Provide an access key of a user with permissions to the Amazon DynamoDB service.
- **Secret Key**: Provide the secret key of the user with the access key provided above.
The Access Key and Secret Key are required properties and must be provided to use the Connector. If these properties are not set, the Connector will not be able to authenticate with the [DynamoDB Service](https://aws.amazon.com/dynamodb/).
```